docs/pass-through.md

```diff
@@ -0,0 +1,237 @@
+# Running Antrea In Pass-Through Mode
+
+Antrea supports chaining with other CNI implementations such as EKS, AKS. In this mode, 
+Antrea enforces k8s network policies, and delegates Pod IP management
+and network connectivity to the primary CNI.
+
+The pass-through mode is designed to be compatible with all major CNI implementations, 
+with emphasis to support cloud provider managed clusters such asAKS, EKS, or potentially
+some flavors of GKE clusters.  
+
+## Deploy and Test Antrea In Cloud 
+Assuming you already have a cloud managed k8s cluster, and have KUBECONFIG environment
+variable set to configuration file of that k8s cluster. You may also refer to [here](cloud.md)
+to create an AKS or EKS cluster on the flight.
+
+Create an Antrea manifest for pass-through mode, for instance
+```bash
+./hack/generate-manifest.sh --encap-mode passThrough > ~/my-yaml/antrea-passthrough.yaml
```

Comment:
I think we should generate this yaml file with make manifest, merge it to repo (like antrea.yml and antrea-ipsec.yml), and publishes it for every release. If different clouds we supported require different yamls, we should generate, merge, and publish those yamls (e.g. antrea-eks.yml, etc.).

Reply:
will do.